vhdbuilder/packer/install-dependencies.sh

```diff
@@ -161,14 +161,24 @@ echo "VHD will be built with containerd as the container runtime"
 updateAptWithMicrosoftPkg
 containerd_manifest="$(jq .containerd manifest.json)" || exit $?
 
-installed_version="$(echo ${containerd_manifest} | jq -r '.edge')"
+containerd_installed_version="$(echo ${containerd_manifest} | jq -r '.edge')"
 if [ "${UBUNTU_RELEASE}" == "18.04" ]; then
-  installed_version="$(echo ${containerd_manifest} | jq -r '.pinned."1804"')"
+  containerd_installed_version="$(echo ${containerd_manifest} | jq -r '.pinned."1804"')"
 fi
 
-containerd_version="$(echo "$installed_version" | cut -d- -f1)"
-containerd_patch_version="$(echo "$installed_version" | cut -d- -f2)"
-installStandaloneContainerd ${containerd_version} ${containerd_patch_version}
+containerd_version="$(echo "$containerd_installed_version" | cut -d- -f1)"
```

Comment:
this naming is more accurate, but to my point above, cutting "containerd_version" from "containerd_installed_version" seems slightly weird